content/en/docs/reference/using-api/deprecation-policy.md

```diff
@@ -365,6 +365,43 @@ This applies only to significant, user-visible behaviors which impact the
 correctness of applications running on Kubernetes or that impact the
 administration of Kubernetes clusters, and which are being removed entirely.
 
+An exception to the above rule is **feature-gates**. Feature gates are key=value 
+pairs that allow for users to enable/disable experimental features.
+
+Feature gates are intended to cover the development life cycle of a feature - they 
+are not intended to be long-term APIs. As such, they are expected to be deprecated 
+and removed after a feature becomes GA or is dropped. 
+
+As a feature moves through the stages, the associated feature-gate will evolve 
+at the same time as announcing its deprecation. In general the life cycle is:
+
+  * The feature is Alpha - its feature gate is disabled by default and can be 
+  enabled by the user.
+  * The feature becomes Beta - its gate becomes enabled by default and can be 
+  disabled by the user.
+  * The feature becomes GA - its gate becomes a no-op and is deprecated.
+  * The feature becomes GA and a deprecation window is announced (see rule below) - its 
+  gate is removed and is no longer accepted.
+
+A feature might be EOL'ed at any point in the life cyle prior to GA.  In that case, 
+the associated feature-gate is deprecated. 
+
+In some cases, the old behavior requires a considerable amount of time before it can 
+be removed. In these cases the feature gate's behavior can be retained until the old 
+behavior is removed, at which point the feature gate can be deprecated as stated above. 
+The gate can still be accepted but should not have any effect and should not cause any 
+errors.
+
+Feature gates are not versioned in the same way as the previously discussed components,
+therefore the rules for deprecation are as follows:
+
+**Rule #8: Feature gates must be deprecated when the corresponding feature they control 
+transitions a lifecycle stage as follows. They must function for no less than:**
```

Comment:
very optional nit: "at least" is slightly more readable than "no less than"

Reply:
Hello, the rules in the full document follow this structure. I am ok to change it after consulting docs. However I believe I should do it outside of the context of this PR :) hope it is ok with you.

Reply:
That's my fault.  If you want to fix it, I could probably live with that :)  In a different PR, of course :)